vignettes/data_protocol.Rmd

```diff
@@ -96,29 +101,29 @@ The distributions currently included in the literature search for each pathogen/
 
-1. **Post hoc removal**: if any `epiparameter` parameters are later identified as being inappropriate then they can be removed from the database. In most cases this is unlikely as limitations can be appended onto data entries to make users aware of limitations (e.g. around assumptions used to infer the distirbution), only in the most extreme cases will data be completely removed from the database.
+1. **Post hoc removal**: if any `{epiparameter}` parameters are later identified as being inappropriate then they can be removed from the database. In most cases this is unlikely as limitations can be appended onto data entries to make users aware of limitations (e.g. around assumptions used to infer the distirbution), only in the most extreme cases will data be completely removed from the database.
 
 Note: systematic reviews focusing on effect sizes can be subject to publication bias (e.g. more positive or significant results in the literature). However, distribution inference does not focus on significance testing or effect sizes, so this bias is not considered in the collection process.
 
 ## Guide to extracting parameters
 
-**Extracting parameters**: for any underlying distributions (e.g. gamma, lognormal), parameters (e.g. shape/scale, meanlog/sdlog), and summary statistics (e.g. mean, standard deviation, median, range or quantiles) given in the paper, values should be recorded verbatim from the paper into the database. When these are read into R using the `epiparameter` package, other aspects of the distribution are automatically calculated and available. For example if the mean and standard deviation of a gamma distribution is reported for a serial interval these values will be stored in the database. But once in R, the shape and scale parameters of the gamma distribution, as well as the confidence intervals, will be automatically reconstructed and the resulting distribution available for use.
+**Extracting parameters**: for any underlying distributions (e.g. gamma, lognormal), parameters (e.g. shape/scale, meanlog/sdlog), and summary statistics (e.g. mean, standard deviation, median, range or quantiles) given in the paper, values should be recorded verbatim from the paper into the database. When these are read into R using the `{epiparameter}` package, other aspects of the distribution are automatically calculated and available. For example if the mean and standard deviation of a gamma distribution is reported for a serial interval these values will be stored in the database. But once in R, the shape and scale parameters of the gamma distribution, as well as the confidence intervals, will be automatically reconstructed and the resulting distribution available for use.
 
-The minimal dataset that is required to be included in the `epiparameter` library is:
+The minimal dataset that is required to be included in the `{epiparameter}` library is:
```

Comment:
Here I would emphasise that this should be done according to what is said explicitly mentioned in the papers

Reply:
The database should reflect the literature, do not include information that is not included in the study,